api/src/main/java/org/sanelib/ils/api/dto/binderOrder/BinderOrderDto.java

```diff
@@ -0,0 +1,118 @@
+package org.sanelib.ils.api.dto.binderOrder;
+
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.apache.commons.lang3.builder.ToStringStyle;
+import org.sanelib.ils.api.dto.DtoWithId;
+import org.sanelib.ils.api.dto.DtoWithLibraryId;
+
+public class BinderOrderDto implements DtoWithId, DtoWithLibraryId {
+
+    private String libraryId;
+    private String id;
+    private String binderId;
+    private String orderDate;
+    private String dueDate;
+    private String returnedDate;
+    private String formLetterNo;
+    private String subject;
+    private String content;
+    private boolean mailStatus;
+    private boolean printStatus;
```

Comment:
We are exposing every attributes as String data type. mailStatusand and printStatus should have String data type instead of boolean just like AuthorDto.contract.